cli/lsp/registries.rs

```diff
@@ -481,7 +480,7 @@ impl ModuleRegistry {
         file_fetcher
         .fetch_with_options(FetchOptions {
           specifier: &specifier,
-          permissions: &PermissionsContainer::allow_all(),
+          permissions: None,
```

Comment:
Yeah - using `None` instead of `PermissionsContainer::allow_all()` is a lot less descriptive and then we're gonna have to carry over a list of available hosts when we implement https://github.com/denoland/deno/pull/25469 anyway.

Reply:
I updated this to use a new enum here that has the permissions container in one of the variants and allow all in the other to make this clear.